src/collective/fingerpointing/locales/de/LC_MESSAGES/collective.fingerpointing.po

```diff
@@ -49,39 +50,39 @@ msgstr "Finger Pointing"
 
 #: collective/fingerpointing/interfaces.py:44
 msgid "If plone.app.iterate is installed, log events like content editing checkouts and checkins."
-msgstr "Wenn Arbeitskopien (plone.app.iterate) aktiviert sind, checkouts und checkins loggen"
+msgstr "Wenn Arbeitskopien (plone.app.iterate) aktiviert sind, dann protokolliere Bearbeitungsereignisse wie Checkouts und Checkins."
 
 #: collective/fingerpointing/profiles.zcml:15
 msgid "Keep track of different events and write them down to an audit log."
 msgstr "Protokolliert relevante Ereignisse in einem Aktionsprotkoll (audit.log)"
 
 #: collective/fingerpointing/interfaces.py:18
 msgid "Log authentication events like users logging in and loggin out."
-msgstr "Protokolliert Ereignisse der Benutzerverwaltung (An- und Abmeldung, Bentuzer hinzufügen oder löschen)."
+msgstr "Protokolliert Ereignisse der Benutzerverwaltung (An- und Abmeldung, Benutzer hinzufügen oder löschen)."
 
 #: collective/fingerpointing/interfaces.py:25
 msgid "Log content type life cycle events like object creation, modification and removal."
-msgstr "Protokolliert den Ereignisse im Lebenszyklus von Inhalten (Erzeugen, Bearbeiten, Löschen)"
+msgstr "Protokolliert die Ereignisse im Lebenszyklus von Inhalten, wie Erstellen, Bearbeiten und Löschen."
 
 #: collective/fingerpointing/interfaces.py:38
 msgid "Log registry events like records being modified."
-msgstr "Protokolliert Änderungen der Konfigurationseinträge (plone.app.registry)."
+msgstr "Protokolliert Änderungen der Konfigurationseinträge (Registry)."
 
 #: collective/fingerpointing/interfaces.py:32
 msgid "Log workflow transitions like object publishing."
-msgstr ""
+msgstr "Protokolliert Arbeitsablauf Übergänge, wie z.B. das Publizieren von Inhalten."
 
 #: collective/fingerpointing/browser/logview.pt:26
 msgid "No audit log file specified; audit log view is disabled."
-msgstr ""
+msgstr "Keine Protokoll-Datei angegeben, die Aktionsprotokoll Ansicht ist deaktiviert."
 
 #: collective/fingerpointing/controlpanel.py:13
 msgid "Settings for the collective.fingerpointing package"
-msgstr "Einstellungen für collective.fingerpointing"
+msgstr "Einstellungen für das collective.fingerpointing Paket"
```

Comment:
having a look at prefs_install_products_form it looks like "Erweiterung" is more common
in addition there are several profiles w/o package or addon in their description.
"Optionale Aktualisierungsunterstützung für plone.session"
i'd leave this one as it was, or go with
"Einstelllungen für die Erweiterung collective.fingerpointing"

Reply:
ack, I will change it.